avocado/utils/memory.py

```diff
@@ -37,17 +37,32 @@ class MemError(Exception):
     pass
 
 
+def get_blk_string(block):
+    """
+    Format the given block id to string
+
+    :param block: memory block id or block string.
+    :type string: like 198 or memory198
+    :return: returns string memory198 if id 198 is given
+    :rtype: string
+    """
+    block = [block]
```

Comment:
Why do you turn this string into a list (with the string on it) if you always access the string itself? I'm really confused here.

Reply:
Yes, you are right. converting to list is unnecessary here. It was a added for different reason in old PR, so not valid now. I will fix this